contributingGuides/PROPOSAL_TEMPLATE.md

```diff
@@ -4,6 +4,8 @@
 
 ### What is the root cause of that problem?
 
+### Which commit or pull request is the root cause of the problem?
```

Comment:
cc @mallenexpensify let me know if we're good to go with this change in proposal template